_posts/2018-03-19-newest-updates-for-tax-data-2018

```diff
@@ -13,19 +13,20 @@ tags:
   - Transit Special Tax Areas
 ---
 
-With April being just around the corner it’s tax time again. If you don’t procrastinate and have already done your taxes, you might be interested in knowing just who is getting that hard earned money you make. Well we have data for that.
+With April just around the corner, it’s tax time again. If you don’t procrastinate and you've already done your taxes, you might be interested in knowing just who is getting that hard-earned money you make. 
+
+Well, we have data for that.
 
 Newest Tax Entities and Areas for 2017
-This is the final list of all entities for 2017, these are the incorporated bodies that have the power to levy a local property tax such as school districts, cities, special service districts, even mosquito abatement districts. While these do not represent exact legal boundaries, they are used by USTC Property Tax Division for assessing properties.
-While the entities can overlap, the areas represent individual polygons of intersecting  entities. 
+This is the final 2017 list of all incorporated bodies that have the power to levy a local property tax: school districts, cities, special service districts&mdash;even mosquito abatement districts. While these entities don't represent exact legal boundaries, they are used by the Utah State Tax Commission, Property Tax Division, for assessing properties. The entities can overlap, but the areas represent individual polygons of intersecting entities. 
  
-Newest Sales Tax areas for Quarter 1, 2018
-Sales Tax Areas are created from the municipal boundaries, metro townships, special sales tax districts and the unincorporated areas. They represent the approximate boundaries of Sales Tax Collection Areas in Utah.
+Newest Sales Tax Areas for Quarter 1, 2018
+Sales Tax Areas are created from municipal boundaries, metro townships, special sales tax districts, and unincorporated areas. They represent the approximate boundaries of Sales Tax Collection Areas in Utah.
```

Comment:
Did I understand this correctly? Was it supposed to be a general reference to "municipal boundaries, metro townships..." etc.? With "the municipal boundaries..." I was wondering, "Which municipal boundaries?"